data_prep/dataset.py

```diff
@@ -0,0 +1,30 @@
+import abc
+import torch.utils.data as data
+
+class Dataset(metaclass=abc.ABCMeta):
```

Comment:
You can make this Dataset class inherit from data.Dataset because with Floor's code, the Graph Reuters will also inherit from Dataset and until then it does not really matter I guess.

Reply:
Also didn't we want a superclass for both Reuters with the `prepare_reuters` function? I guess it is not the plan anymore?

Reply:
Yes, I forgot about that. I will add it to this PR